packages/frontend/src/components/AppPicker/styles.module.scss

```diff
@@ -94,8 +94,33 @@
   display: flex;
   flex-direction: column;
   justify-content: center;
+  width: 208px;
   p {
     margin: 2px 0;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+  button {
+    border: none;
+    background: none;
+    color: var(--globalLinkColor);
+    text-align: left;
+    margin: 2px 0;
+    padding: 0;
+  }
+}
+
+.appInfoFullWidth {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 100%;
+  p {
+    margin: 2px 0;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
```

Comment:
This is a bit ugly but solves the problem. I'm open to suggestions on how to make this cleaner. So basically what I want is to force the parent div's width to 208 pixels **when in the app picker**, but in the dialog it can be 100% width.

Reply:
You can remove the redundant .appInfoFullWidth completely and just add these lines:

```
/** limited width in app picker */
.appListItem .appInfo {
  width: 208px;
}
```

and remove the condition in index.tsx

`className={setSearchQuery ? styles.appInfoFullWidth : styles.appInfo} `

of course

Reply:
Just to clarify, the math above doesn't check out because I forgot to mention there are multiple divs involved. Ultimately there's **a** div with hard width set to 300px, which also has some margin:

<img width="221" height="160" alt="image" src="https://github.com/user-attachments/assets/8136d826-ba36-4dbd-98af-5efcf2499aa9" />

So it's really 288 - 64 - 16 = 208.